### PR TITLE
Fix typos accross the whole codebase

### PR DESCRIPTION
**Pull request by Pierre-Sassoulas**
## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :hammer: Refactoring   |
| ✓   | :scroll: Docs          |

## Description

Follow-up to #5558

### CODE REVIEW

File: pylint/checkers/base.py [hunk 1]
```diff
@@ -298,13 +298,13 @@ def _get_break_loop_node(break_node):
 
 def _loop_exits_early(loop):
     """
-    Returns true if a loop may ends up in a break statement.
+    Returns true if a loop mays end up in a break statement.
```
Comment (DanielNoord) [inline pylint/checkers/base.py:301]:
```suggestion
    Returns true if a loop may end with a break statement.
```

File: pylint/checkers/base.py [hunk 2]
```diff
@@ -298,13 +298,13 @@ def _get_break_loop_node(break_node):
...
 
     Args:
         loop (astroid.For, astroid.While): the loop node inspected.
 
     Returns:
-        bool: True if the loop may ends up in a break statement, False otherwise.
+        bool: True if the loop mays end up in a break statement, False otherwise.
```
Comment (DanielNoord) [inline pylint/checkers/base.py:307]:
```suggestion
        bool: True if the loop may end with a break statement, False otherwise.
```

File: pylint/checkers/base.py [hunk 3]
```diff
@@ -1359,7 +1359,7 @@ def is_iterable(internal_node):
     def visit_return(self, node: nodes.Return) -> None:
         """1 - check is the node has a right sibling (if so, that's some
         unreachable code)
-        2 - check is the node is inside the finally clause of a try...finally
+        2 - check is the node is inside the 'finally' clause of a try...finally
```
Comment (DanielNoord) [inline pylint/checkers/base.py:1362]:
```suggestion
        2 - check if the node is inside the 'finally' clause of a try...finally
```

File: pylint/checkers/base.py [hunk 4]
```diff
@@ -1377,7 +1377,7 @@ def visit_continue(self, node: nodes.Continue) -> None:
     def visit_break(self, node: nodes.Break) -> None:
         """1 - check is the node has a right sibling (if so, that's some
```
Comment (DanielNoord) [inline pylint/checkers/base.py:1378]:
```suggestion
        """1 - check if the node has a right sibling (if so, that's some
```

File: pylint/checkers/base.py [hunk 5]
```diff
@@ -1377,7 +1377,7 @@ def visit_continue(self, node: nodes.Continue) -> None:
...
         unreachable code)
-        2 - check is the node is inside the finally clause of a try...finally
+        2 - check is the node is inside the 'finally' clause of a try...finally
```
Comment (DanielNoord) [inline pylint/checkers/base.py:1380]:
```suggestion
        2 - check if the node is inside the 'finally' clause of a try...finally
```

Comment (DanielNoord) [reply pylint/checkers/base.py:1380]: You added quotes to other occurrences of `try...finally`. Do you want to do that here as well?

File: pylint/checkers/base.py [hunk 6]
```diff
@@ -1490,14 +1490,14 @@ def _check_unreachable(self, node):
             self.add_message("unreachable", node=unreach_stmt)
 
     def _check_not_in_finally(self, node, node_name, breaker_classes=()):
-        """check that a node is not inside a finally clause of a
-        try...finally statement.
-        If we found before a try...finally block a parent which its type is
+        """check that a node is not inside a 'finally' clause of a
+        'try...finally' statement.
+        If we found before a 'try...finally' block a parent which its type is
```
Comment (DanielNoord) [inline pylint/checkers/base.py:1495]: This sentences doesn't work.

File: pylint/checkers/base.py [hunk 7]
```diff
@@ -1490,14 +1490,14 @@ def _check_unreachable(self, node):
...
         in breaker_classes, we skip the whole check."""
         # if self._tryfinallys is empty, we're not an in try...finally block
         if not self._tryfinallys:
             return
-        # the node could be a grand-grand...-children of the try...finally
+        # the node could be a grand-grand...-children of the 'try...finally'
```
Comment (DanielNoord) [inline pylint/checkers/base.py:1500]:
```suggestion
        # the node could be a grand-grand...-childr of the 'try...finally'
```

File: pylint/checkers/base.py [hunk 8]
```diff
@@ -1613,7 +1613,7 @@ def _check_self_assigning_variable(self, node):
             if not isinstance(target, nodes.AssignName):
                 continue
             if isinstance(scope, nodes.ClassDef) and target.name in scope_locals:
-                # Check that the scope is different than a class level, which is usually
+                # Check that the scope is different from a class level, which is usually
```
Comment (DanielNoord) [inline pylint/checkers/base.py:1616]: Not sure about this change. Doesn't the `if` statement make sue that scope is indeed a class level?

File: pylint/checkers/base.py [hunk 9]
```diff
@@ -2218,7 +2218,7 @@ def _check_docstring(
         report_missing=True,
         confidence=interfaces.HIGH,
     ):
-        """check the node has a non empty docstring"""
+        """check the node has a non-empty docstring"""
```
Comment (DanielNoord) [inline pylint/checkers/base.py:2221]:
```suggestion
        """Check if the node has a non-empty docstring"""
```

File: pylint/checkers/design_analysis.py [hunk 10]
```diff
@@ -636,7 +636,7 @@ def visit_if(self, node: nodes.If) -> None:
     def _check_boolean_expressions(self, node):
         """Go through "if" node `node` and counts its boolean expressions
```
Comment (DanielNoord) [inline pylint/checkers/design_analysis.py:637]:
```suggestion
        """Go through "if" node `node` and count its boolean expressions
```

File: pylint/checkers/design_analysis.py [hunk 11]
```diff
@@ -636,7 +636,7 @@ def visit_if(self, node: nodes.If) -> None:
...
 
-        if the "if" node test is a BoolOp node
+        if the 'if' node test is a BoolOp node
```
Comment (DanielNoord) [inline pylint/checkers/design_analysis.py:639]: Can we remove L638?

File: pylint/checkers/refactoring/recommendation_checker.py [hunk 12]
```diff
@@ -337,7 +337,7 @@ def visit_const(self, node: nodes.Const) -> None:
 
     def _detect_replacable_format_call(self, node: nodes.Const) -> None:
         """Check whether a string is used in a call to format() or '%' and whether it
-        can be replaced by a f-string"""
+        can be replaced by an f-string"""
```
Comment (DanielNoord) [inline pylint/checkers/refactoring/recommendation_checker.py:340]: I'm doubting myself here, but should this be `an`?

File: pylint/checkers/strings.py [hunk 13]
```diff
@@ -517,8 +517,8 @@ def _check_new_format(self, node, func):
                     # Verify the required number of positional arguments
                     # only if the .format got at least one keyword argument.
                     # This means that the format strings accepts both
-                    # positional and named fields and we should warn
-                    # when one of the them is missing or is extra.
+                    # positional and named fields, and we should warn
```
Comment (DanielNoord) [inline pylint/checkers/strings.py:520]:
```suggestion
                    # positional and named fields and we should warn
```

File: pylint/checkers/strings.py [hunk 14]
```diff
@@ -973,15 +973,14 @@ def _is_long_string(string_token: str) -> bool:
 def _get_quote_delimiter(string_token: str) -> str:
     """Returns the quote character used to delimit this token string.
 
-    This function does little checking for whether the token is a well-formed
-    string.
+    This function check whether the token is a well-formed string.
```
Comment (DanielNoord) [inline pylint/checkers/strings.py:976]:
```suggestion
    This function checks whether the token is a well-formed string.
```

File: pylint/checkers/typecheck.py [hunk 15]
```diff
@@ -464,8 +464,8 @@ def _emit_no_member(
 
     The following cases are ignored:
 
-        * the owner is a function and it has decorators.
-        * the owner is an instance and it has __getattr__, __getattribute__ implemented
+        * the owner is a function, and it has decorators.
```
Comment (DanielNoord) [inline pylint/checkers/typecheck.py:467]:
```suggestion
        * the owner is a function and it has decorators.
```

File: pylint/checkers/typecheck.py [hunk 16]
```diff
@@ -464,8 +464,8 @@ def _emit_no_member(
...
+        * the owner is an instance, and it has __getattr__, __getattribute__ implemented
```
Comment (DanielNoord) [inline pylint/checkers/typecheck.py:468]:
```suggestion
        * the owner is an instance and it has __getattr__, __getattribute__ implemented
```

File: pylint/checkers/typecheck.py [hunk 17]
```diff
@@ -1058,7 +1058,7 @@ def visit_attribute(self, node: nodes.Attribute) -> None:
                 continue
             except astroid.NotFoundError:
                 # This can't be moved before the actual .getattr call,
-                # because there can be more values inferred and we are
+                # because there can be more values inferred, and we are
```
Comment (DanielNoord) [inline pylint/checkers/typecheck.py:1061]:
```suggestion
                # because there can be more values inferred and we are
```

File: pylint/config/find_default_config_files.py [hunk 18]
```diff
@@ -71,7 +71,7 @@ def find_default_config_files() -> Iterator[str]:
 
 
 def find_pylintrc() -> Optional[str]:
-    """search the pylint rc file and return its path if it find it, else None"""
+    """search the pylint rc file and return its path if it finds it, else None"""
```
Comment (DanielNoord) [inline pylint/config/find_default_config_files.py:74]:
```suggestion
    """Search the pylint rc file and return its path if it finds it, else return None"""
```

File: pylint/config/man_help_formatter.py [hunk 19]
```diff
@@ -4,10 +4,10 @@
 import optparse  # pylint: disable=deprecated-module
 import sys
 import time
+from abc import ABCMeta
 
 
-# pylint: disable=abstract-method; by design?
-class _ManHelpFormatter(optparse.HelpFormatter):
+class _ManHelpFormatter(optparse.HelpFormatter, metaclass=ABCMeta):
```
Comment (DanielNoord) [inline pylint/config/man_help_formatter.py:10]: This shouldn't really be in a `typo` PR.. Perhaps do a quick separate PR?

File: pylint/config/option_manager_mixin.py [hunk 20]
```diff
@@ -282,7 +282,7 @@ def read_config_file(self, config_file=None, verbose=None):
                 # Use this encoding in order to strip the BOM marker, if any.
                 with open(config_file, encoding="utf_8_sig") as fp:
                     parser.read_file(fp)
-                # normalize sections'title
+                # normalize section's title
```
Comment (DanielNoord) [inline pylint/config/option_manager_mixin.py:285]:
```suggestion
                # normalize each section's title
```

`_sections` is plural

File: pylint/epylint.py [hunk 21]
```diff
@@ -67,8 +67,8 @@
 
 
 def _get_env():
-    """Extracts the environment PYTHONPATH and appends the current sys.path to
-    those."""
+    """Extracts the environment PYTHONPATH and appends the current 'sys.path'
+    to those."""
```
Comment (DanielNoord) [inline pylint/epylint.py:71]:
```suggestion
    to it."""
```

File: pylint/extensions/code_style.py [hunk 22]
```diff
@@ -26,7 +26,7 @@ class CodeStyleChecker(BaseChecker):
        i.e. detect a common issue or improve performance
        => it should probably be part of the core checker classes
     2. Is it something that would improve code consistency,
-       maybe because it's slightly better with regards to performance
+       maybe because it's slightly better regarding performance
```
Comment (DanielNoord) [inline pylint/extensions/code_style.py:29]:
```suggestion
       maybe because it's slightly better with regards to performance
```

File: pylint/lint/parallel.py [hunk 23]
```diff
@@ -78,7 +78,7 @@ def _worker_check_single_file(
 
 def _merge_mapreduce_data(linter, all_mapreduce_data):
     """Merges map/reduce data across workers, invoking relevant APIs on checkers"""
-    # First collate the data, preparing it so we can send it to the checkers for
+    # First collate the data, preparing it, so we can send it to the checkers for
```
Comment (DanielNoord) [inline pylint/lint/parallel.py:81]:
```suggestion
    # First collate the data and prepare it, so we can send it to the checkers for
```

File: pylint/lint/run.py [hunk 24]
```diff
@@ -46,7 +46,7 @@ def cb_list_confidence_levels(option, optname, value, parser):
 
 
 def cb_init_hook(optname, value):
-    """exec arbitrary code to set sys.path for instance"""
+    """Exec arbitrary code to set 'sys.path' for instance"""
```
Comment (DanielNoord) [inline pylint/lint/run.py:49]:
```suggestion
    """Execute arbitrary code to set 'sys.path' for instance"""
```

File: pylint/testutils/output_line.py [hunk 25]
```diff
@@ -93,8 +93,8 @@ def from_msg(cls, msg: Message, check_endline: bool = True) -> "OutputLine":
 
     @staticmethod
     def _get_column(column: str) -> int:
-        """Handle column numbers with the exception of pylint < 3.8 not having them
-        in the ast parser.
+        """Handle column numbers except for python < 3.8 not having them in
+        the ast parser.
```
Comment (DanielNoord) [inline pylint/testutils/output_line.py:97]:
```suggestion
        """Handle column numbers except for python < 3.8. The ast parser in those versions does 
        return them.
```

File: tests/message/unittest_message.py [hunk 26]
```diff
@@ -11,13 +11,13 @@
 
 def test_new_message(message_definitions: ValuesView[MessageDefinition]) -> None:
     def build_message(
-        message_definition: MessageDefinition, location_value: MessageLocationTuple
+        message_definition_: MessageDefinition, location_value: MessageLocationTuple
```
Comment (DanielNoord) [inline tests/message/unittest_message.py:14]: ?
Is this interfering with any other variable?

File: tests/test_regr.py [hunk 27]
```diff
@@ -51,9 +51,8 @@ def disable():
 
 @pytest.fixture
 def finalize_linter(linter: PyLinter) -> Iterator[PyLinter]:
-    """call reporter.finalize() to cleanup
-    pending messages if a test finished badly
-    """
+    """Call reporter.finalize() to clean up pending messages if a test
+    finished badly."""
```
Comment (DanielNoord) [inline tests/test_regr.py:55]:
```suggestion
    finished badly.
    """
```

This is a PEP8 recommendation.

**Review by DanielNoord — CHANGES_REQUESTED** [review]
I don't think I have caught everything so I'll do a new review after these changes have been merged. It is quite difficult to keep focused on spelling for such long PRs 😄

File: pylint/checkers/refactoring/recommendation_checker.py (hunk 12 above)
Comment (jacobtylerwalls) [reply pylint/checkers/refactoring/recommendation_checker.py:340]: Yes, I would say "an eff string"

File: pylint/checkers/base.py (hunk 5 above)
Comment (Pierre-Sassoulas) [reply pylint/checkers/base.py:1380]: Nice catch, it was not detected by my grammer linter :)
```suggestion
        2 - check if the node is inside the 'finally' clause of a 'try...finally'
```

File: pylint/checkers/design_analysis.py [hunk 28]
```diff
@@ -636,7 +636,7 @@ def visit_if(self, node: nodes.If) -> None:
     def _check_boolean_expressions(self, node):
         """Go through "if" node `node` and counts its boolean expressions
```
Comment (Pierre-Sassoulas) [inline pylint/checkers/design_analysis.py:638]:
```suggestion
```

File: pylint/checkers/refactoring/recommendation_checker.py (hunk 12 above)
Comment (Pierre-Sassoulas) [reply pylint/checkers/refactoring/recommendation_checker.py:340]: Thougth the same as @jacobtylerwalls, it's a tricky one :smile:

File: pylint/config/man_help_formatter.py (hunk 19 above)
Comment (Pierre-Sassoulas) [reply pylint/config/man_help_formatter.py:10]: I got carried away, sorry :smile:

File: pylint/pyreverse/diagrams.py [hunk 29]
```diff
@@ -16,9 +16,6 @@
 # Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
 # For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
```
[the remark this answers is not in the document]
Comment (Pierre-Sassoulas) [reply pylint/pyreverse/diagrams.py:18]: We're in a module named "diagram", isn't that docstring just noise ?

File: tests/message/unittest_message.py (hunk 26 above)
Comment (Pierre-Sassoulas) [reply tests/message/unittest_message.py:14]: It's shadowing an outer variable (I got carried away)

File: pylint/checkers/base.py (hunk 6 above)
Comment (Pierre-Sassoulas) [reply pylint/checkers/base.py:1495]: Right. I did not read it, just added `'`. guess we have to fix it now.
```
        If we found a parent which type is in breaker_classes before
        a 'try...finally' block we skip the whole check.
```
?

File: pylint/checkers/base.py (hunk 8 above)
Comment (Pierre-Sassoulas) [reply pylint/checkers/base.py:1616]: Maybe the comment should be moved before the if ?

File: pylint/pyreverse/diagrams.py (hunk 29 above)
[the remark this answers is not in the document]
Comment (DanielNoord) [reply pylint/pyreverse/diagrams.py:18]: Hmm perhaps. As you know I think docstrings (almost) never hurt. But I'll leave this one to you!

File: pylint/checkers/base.py (hunk 6 above)
Comment (DanielNoord) [reply pylint/checkers/base.py:1495]: Seems good to me!

File: pylint/checkers/base.py (hunk 8 above)
Comment (DanielNoord) [reply pylint/checkers/base.py:1616]: That could be it yeah!

File: pylint/checkers/refactoring/recommendation_checker.py (hunk 12 above)
Comment (DanielNoord) [reply pylint/checkers/refactoring/recommendation_checker.py:340]: Ah yeah! My bad, thanks @jacobtylerwalls!

File: pylint/checkers/base.py [hunk 30]
```diff
@@ -1357,9 +1357,9 @@ def is_iterable(internal_node):
 
     @utils.check_messages("unreachable", "lost-exception")
     def visit_return(self, node: nodes.Return) -> None:
-        """1 - check is the node has a right sibling (if so, that's some
+        """1 - check if the node has a right sibling (if so, that's some
         unreachable code)
-        2 - check is the node is inside the finally clause of a try...finally
+        2 - check if the node is inside the 'finally' clause of a try...finally
```
Comment (DanielNoord) [inline pylint/checkers/base.py:1362]:
```suggestion
        2 - check if the node is inside the 'finally' clause of a 'try...finally'
```

File: pylint/checkers/base.py [hunk 31]
```diff
@@ -1490,14 +1490,14 @@ def _check_unreachable(self, node):
             self.add_message("unreachable", node=unreach_stmt)
 
     def _check_not_in_finally(self, node, node_name, breaker_classes=()):
-        """check that a node is not inside a finally clause of a
-        try...finally statement.
-        If we found before a try...finally block a parent which its type is
+        """check that a node is not inside a 'finally' clause of a
+        'try...finally' statement.
+        If we found before a 'try...finally' block a parent which its type is
         in breaker_classes, we skip the whole check."""
```
Comment (DanielNoord) [inline pylint/checkers/base.py:1496]:
```suggestion
        If we find a parent which type is in breaker_classes before 
        a 'try...finally' block we skip the whole check."""
```

File: pylint/config/man_help_formatter.py (hunk 19 above)
Comment (DanielNoord) [reply pylint/config/man_help_formatter.py:10]: Are you going to make a separate PR for this? Or should I add a suggestion to revert it so you can apply it?

File: pylint/testutils/output_line.py [hunk 32]
```diff
@@ -93,8 +93,8 @@ def from_msg(cls, msg: Message, check_endline: bool = True) -> "OutputLine":
 
     @staticmethod
     def _get_column(column: str) -> int:
-        """Handle column numbers with the exception of pylint < 3.8 not having them
-        in the ast parser.
+        """Handle column numbers except for python < 3.8. The ast parser in those versions does
```
Comment (DanielNoord) [inline pylint/testutils/output_line.py:96]:
```suggestion
        """Handle column numbers except for python < 3.8. The ast parser in those versions doesn't
```

I suggested poorly 😅

File: tests/message/unittest_message.py (hunk 26 above)
Comment (DanielNoord) [reply tests/message/unittest_message.py:14]: Want to do another PR for this? Or include it in this one?

**Review by DanielNoord — CHANGES_REQUESTED** [review]
I have not re-checked everything (hoping that I was fairly thorough in my first review). I think we're almost there.

File: pylint/checkers/base.py [hunk 33]
```diff
@@ -1613,7 +1613,7 @@ def _check_self_assigning_variable(self, node):
             if not isinstance(target, nodes.AssignName):
                 continue
             if isinstance(scope, nodes.ClassDef) and target.name in scope_locals:
```
Comment (Pierre-Sassoulas) [inline pylint/checkers/base.py:1615]:
```suggestion
                continue
            # Check that the scope is different from a class level, which is usually
            if isinstance(scope, nodes.ClassDef) and target.name in scope_locals:
            
```

File: pylint/checkers/base.py (hunk 8 above)
Comment (Pierre-Sassoulas) [inline pylint/checkers/base.py:1616]:
```suggestion
```

File: pylint/config/man_help_formatter.py (hunk 19 above)
Comment (Pierre-Sassoulas) [reply pylint/config/man_help_formatter.py:10]: I'm going to rebase and separate that.

File: tests/message/unittest_message.py (hunk 26 above)
Comment (Pierre-Sassoulas) [reply tests/message/unittest_message.py:14]: Let's include in this one, I don't want to be responsible for going above 1.5°c of global warming due to an irresponsible use of heavy duty primers :smile:

Comment (DanielNoord) [reply tests/message/unittest_message.py:14]: If my code is correct they shouldn't on a change that only affects `tests/**` but I think this is fine to include here indeed 😄

File: pylint/checkers/utils.py [hunk 34]
```diff
@@ -512,7 +512,7 @@ def parse_format_string(
     format_string: str,
 ) -> Tuple[Set[str], int, Dict[str, str], List[str]]:
     """Parses a format string, returning a tuple of (keys, num_args), where keys
-    is the set of mapping keys in the format string, and num_args is the number
+    are the set of mapping keys in the format string, and num_args is the number
```
Comment (jacobtylerwalls) [inline pylint/checkers/utils.py:515]: NBD but I would have left this as is, `keys` is the variable name.

File: pylint/checkers/base.py [hunk 35]
```diff
@@ -1612,8 +1612,9 @@ def _check_self_assigning_variable(self, node):
                 continue
             if not isinstance(target, nodes.AssignName):
                 continue
+            # Check that the scope is different from a class level, which is usually
```
[the remark this answers is not in the document]
Comment (Pierre-Sassoulas) [reply pylint/checkers/base.py:1615]: Handled during rebase

File: tests/unittest_reporting.py [hunk 36]
```diff
@@ -91,8 +91,8 @@ def test_template_option_end_line(linter) -> None:
     assert out_lines[2] == "my_mod:2:0:2:4: C0301: Line too long (3/4) (line-too-long)"
 
 
-def test_template_option_non_exisiting(linter) -> None:
-    """Test the msg-template option with a non existing options.
+def test_template_option_non_existing(linter) -> None:
+    """Test the msg-template option with a non-existing options.
```
Comment (jacobtylerwalls) [inline tests/unittest_reporting.py:95]: "a options" isn't quite right. maybe: "with nonexistent options"

File: pylint/checkers/utils.py (hunk 34 above)
Comment (Pierre-Sassoulas) [reply pylint/checkers/utils.py]: Thanks, you're right ! 
3e28cb1

File: tests/unittest_reporting.py (hunk 36 above)
Comment (Pierre-Sassoulas) [reply tests/unittest_reporting.py]: b5fc408

File: pylint/checkers/utils.py [hunk 37]
```diff
@@ -512,10 +512,9 @@ def parse_format_string(
     format_string: str,
 ) -> Tuple[Set[str], int, Dict[str, str], List[str]]:
     """Parses a format string, returning a tuple of (keys, num_args), where 'keys'
-    is the set of mapping keys in the format string, and num_args is the number
-    of arguments required by the format string.  Raises
-    IncompleteFormatString or UnsupportedFormatCharacter if a
-    parse error occurs."""
+    is the set of mapping keys in the format string, and 'num_args' is the number
+    of arguments required by the format string. Raises IncompleteFormatString or
+    UnsupportedFormatCharacter if a parse error occurs."""
```
Comment (DanielNoord) [inline pylint/checkers/utils.py:517]:
```suggestion
    UnsupportedFormatCharacter if a parse error occurs.
    """
```

**Review by DanielNoord — APPROVED** [review]
I think it would be good if at least one other person did a full review of this.

**Comment by Pierre-Sassoulas** [comment]
@jacobtylerwalls would you have the time to do a full review,? :)

File: tests/test_func.py [hunk 38]
```diff
@@ -142,8 +142,8 @@ def test_functionality(module_file, messages_file, dependencies, recwarn):
     __test_functionality(module_file, messages_file, dependencies)
     warning = None
     try:
-        # Catch <unknown>:x: DeprecationWarning: invalid escape sequence
-        # so it's not shown during tests
+        # Catch <unknown>:x: DeprecationWarning: invalid escape sequence,
+        # so, it's not shown during tests
```
Comment (jacobtylerwalls) [inline tests/test_func.py:146]: I think this edit was unnecessary, but it's not hurting anything.
```suggestion
        # Catch <unknown>:x: DeprecationWarning: invalid escape sequence
        # so it's not shown during tests
```

File: tests/test_functional.py [hunk 39]
```diff
@@ -74,8 +74,8 @@ def test_functional(
     lint_test.runTest()
     warning = None
     try:
-        # Catch <unknown>:x: DeprecationWarning: invalid escape sequence
-        # so it's not shown during tests
+        # Catch <unknown>:x: DeprecationWarning: invalid escape sequence,
+        # so, it's not shown during tests
```
Comment (jacobtylerwalls) [inline tests/test_functional.py:78]:
```suggestion
        # Catch <unknown>:x: DeprecationWarning: invalid escape sequence
        # so it's not shown during tests
```

File: tests/unittest_reporting.py [hunk 40]
```diff
@@ -91,8 +91,8 @@ def test_template_option_end_line(linter) -> None:
     assert out_lines[2] == "my_mod:2:0:2:4: C0301: Line too long (3/4) (line-too-long)"
 
 
-def test_template_option_non_exisiting(linter) -> None:
-    """Test the msg-template option with a non existing options.
+def test_template_option_non_existing(linter) -> None:
+    """Test the msg-template option with non-existent options.
```
Comment (jacobtylerwalls) [inline tests/unittest_reporting.py:95]:
```suggestion
    """Test the msg-template option with nonexistent options.
```

**Review by jacobtylerwalls — APPROVED** [review]
LGTM. Some tiny suggestions you can ignore.

**Comment by jacobtylerwalls** [comment]
Oh, and for the commit msg, it's "typos", not "typoes" :D

**Comment by Pierre-Sassoulas** [comment]
> , it's "typos", not "typoes" :D

No one caught this before 😄 Thanks for the review ! :)
